aesm-client/src/imp/aesm_protobuf/mod.rs

```diff
@@ -107,4 +109,150 @@ impl AesmClient {
         Ok(QuoteResult::new(quote, qe_report))
     }
 
+
+    ///
+    /// Purpose of following functions is to stay as close as possible to protobuf interface.
+    /// for more usable API see AesmClient in rust-sgx/aesm-client/src/lib.rs
+    /// 
+    
+    ///
+    /// Returns number of keys stored in AESM service. This is used internally in get_supported_att_key_ids.
+    ///
+    pub fn get_supported_att_key_id_num(&self) -> Result<u32> {
```

Comment:
I think this should be inlined in `get_supported_att_key_ids` below

Reply:
inlined